release/supported-portal-versions.txt

```diff
@@ -2,4 +2,4 @@
 7.3
 2023.q3
 2023.q4
-2024.q1
\ No newline at end of file
+2024.q1
```

Comment:
Since the quarterly releases are just related to DXP, does it make sense we include this info here in Portal?

Reply:
The quarterly releases were already there, but yea it does not make sense so I will delete them. Thank you

Reply:
Before I forward your pull, @Wykenz, can you please just remove this part from the first commit?

Thanks